packages/kotti-ui/source/next/kotti-field-radio-group/KtFieldRadioGroup.vue

```diff
@@ -19,13 +19,6 @@
 						field.isDisabled || Boolean(option.isDisabled),
 				}"
 			>
-				<input
```

Comment:
Re-ordered to fix issues with `:not(:first-child)` adding margin to the first visible child